.github/actions/minikube-setup/action.yml

```diff
@@ -8,7 +8,7 @@ runs:
       uses: manusa/actions-setup-minikube@v2.7.2
       with:
         minikube version: 'v1.28.0'
-        kubernetes version: 'v1.24.9'
+        kubernetes version: 'v1.23.15'
```

Comment:
This is revert previous PR which upgraded kube to 1.24 and now sometimes breaks the e2e tests because it was not able to download mlserver image due to https://github.com/manusa/actions-setup-minikube/pull/77